src/main/create-operator-json/operators/recruit.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+import counterType from '../core/counterTypeEnums';
+import operatorId from '../core/operatorIdEnum';
+import Operator from '../core/operatorFactory';
+import r6operators from "r6operators";
+
+let recruit = new Operator(r6operators.recruit_blue, operatorId.recruit, "N/A");
```

Comment:
Recruit's image does not show and he is labelled wrong when you hover over him

Reply:
fixed